tests/inspect.go

```diff
@@ -27,6 +27,8 @@ func Inspect(o *option.Option) {
 			gomega.Expect(image).To(gomega.Equal(defaultImage))
 			containerName := command.StdoutStr(o, "inspect", "--format", "{{.Name}}", testContainerName)
 			gomega.Expect(containerName).To(gomega.Equal(testContainerName))
+			gomega.Expect(command.StdoutStr(o, "inspect", "--format", "{{.State.Status}}", testContainerName)).To(gomega.Equal("exited"))
```

Comment:
Should we do case insensitive check here?

Reply:
I think it should be case sensitive for strict stdout checking. The string is also hardcoded in upstream.

Reply:
The reasoning behind case insensitive comparison is to make the test flexible enough to accommodate minor case text changes in the upstream without breaking our functional test. It is not uncommon, developer changes casing of hardcoded text which are meant for display purpose. For example, if a developer changes the hardcoded value from "exited" to "Exited", our test case will fail where as the functionality wise it is same.

Reply:
As this is stdout instead of stderr, it can be programatically consumed by downstream. Changing from "exited" to "Exited" will break their workflow. So we need to do the case sensitive check to protect downstream's workflow. If upstream changes to "Exited", we can at least identify it from the failing tests. We will need to change the tests, and execute the breaking change process to notify downstream.

Reply:
General practice of comparing std out in such cases is usually to compare as case insensitive to make it more resilient. The E2E test is to check the functionality not to ensure every output text especially the casing of the msg. As i mentioned earlier, changing the text from "exited" to "Exited" does not change the functionality but the output format. I still dont think, we need to safeguard the casing of the msg in our e2e test. However, since you have a strong preference on keeping it as it is, i am approving the PR but my recommendation is still same.

Reply:
I currently we have some downstream trying to parse the stdout so there is a risk of breaking their workflow. 

Overall the chance case change in upstream is very small. But when it happens, if we have strict validation, we can catch it and  publish a super quick PR to fix the tests, then notify downstream by other processes. If we don't have the strict validation, downstream customers will take the pain.

In either of the scenarios, we need to communicate with downstream customers. So it we have strict validation, it only takes us 5 mins to fix tests. Otherwise, it will hurt downstream customer's trust. Trust takes years to build, seconds to break, and forever to repair. In my opinion, even under same possibility, hurting trust even with a small breaking change   can have a much deeper impact to Finch than just a 5-min test fix.